src/converse-muc-views.js

```diff
@@ -853,12 +853,15 @@ converse.plugins.add('converse-muc-views', {
             },
 
             parseMessageForCommands (text) {
-                if (_converse.muc_disable_moderator_commands) {
+                if (_converse.muc_disable_moderator_commands && _converse.muc_disable_moderator_commands.constructor !== Array) {
                     return _converse.ChatBoxView.prototype.parseMessageForCommands.apply(this, arguments);
                 }
                 const match = text.replace(/^\s*/, "").match(/^\/(.*?)(?: (.*))?$/) || [false, '', ''],
                       args = match[2] && match[2].splitOnce(' ').filter(s => s) || [],
                       command = match[1].toLowerCase();
+                if (_.includes(_converse.muc_disable_moderator_commands, command)) {
```

Comment:
It's not guaranteed that `_converse.muc_disable_moderator_commands` is an array here.

I'd rather do something like this:

```const disabled_commands = _.isArray(_converse.muc_disable_moderator_commands) : _converse.muc_disable_moderator_commands ? [];```

And then use that here and further down below. Then you don't need to check if it's an array there either.